folsom-semantic-metrics/pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>com.spotify</groupId>
     <artifactId>folsom-parent</artifactId>
-    <version>1.3.1-SNAPSHOT</version>
+    <version>2.0.0-SNAPSHOT</version>
```

Comment:
Since I'm changing the interfaces perhaps I need to bump the major version? Not sure if this is overkill or not.